src/tests/roc_audio/test_depacketizer.cpp

```diff
@@ -25,7 +25,8 @@ namespace audio {
 
 namespace {
 
-enum { MaxBufSize = 4000, SamplesPerPacket = 200, NumCh = 2, ChMask = 0x3 };
+enum { MaxBufSize = 4000, SamplesPerPacket = 200, SampleRate=100, ChMask = 0x3,
+       SamplesSize = 400 };
```

Comment:
`SamplesSize` is `SamplesPerPacket * <number of channels>`, right? If so, I suggest to explicitly define NumCh and define SamplesSize as `SamplesPerPacket * NumCh`, to make it clear. I think it's OK that we'll have both ChMask and NumCh defined, since they will be close to each other.